components/automate-chef-io/content/docs/desktop.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Desktop"
+description = "Using the Desktop View"
+date = 2018-03-26T16:01:47-07:00
+draft = false
+bref = ""
+toc = true
++++
+
+## Overview
+The _Desktop_ page displays information about all desktops connected to Chef Automate. It can be found via the top nav of your Automate instance, at `automate-url/desktop`
+Desktops appear in this view after a Chef Infra Client run has executed.
+
+### Daily Check In
+Shows a count of desktops that have and are reporting in to Automate. Desktops that have not checked in in the last twenty-four hours will be counted as `unknown`.
+Selecting a status filter opens the list view of the desktops (Insight) with the selected status.
+
+### Check In History
+Shows the history of checked-in and unknown desktop counts over time. Timescale can be adjusted using the timescale dropdown menu.
+
+### Top Errors
+Shows the most common errors that have occurred across all desktops over the last twenty-four hours, and a count of the machines that incurred the error.
+Selecting an error opens the list view of the desktops (Insight) with the error to enable further investigation.
+
+### Unknown Duration
+Shows count of desktops that have been `unknown` (have not reported in to Automate) for the given time frames.
+
+### Insight (Desktop List)
+Shows a list of desktops. Available filters found at the top of the list. Additional filtering attributes may be selected using the `Choose Attributes` dropdown menu.
+
+### Node Detail
+Shows details of a single desktop and a graph displaying the latest status per day for the desktop (failed, converged, or unknown).
+
+### Configuration
+The Automate install for the desktop experience requires the use of a flag, `--with-desktop`.
+When starting Automate with the desktop flag, data lifecycle configurations will be set to never mark nodes as missing and never delete missing nodes. Users are encouraged to not modify these settings.
```

Comment:
whoa, i prolly dont know enough. but i feel like support has this push right now to get everyone to set these. are folks not going to run into space issues with these settings?

Reply:
I'm just getting started on the Desktop docs. I'll confirm what our base case is.

Reply:
@susanev are you thinking about retention settings? this is slightly different -about nodes going missing/deleting missing nodes. the default config for that is to mark as missing after 1 day, i believe, and then delete if missing a week. this shouldn’t affect space negatively. the presumption is that desktops aren’t ephemeral, so the whole “missing” thing for a desktop is actually “unknown”

Reply:
@susanev are you thinking about retention settings? this is slightly different -about nodes going missing/deleting missing nodes. the default config for that is to mark as missing after 1 day, i believe, and then delete if missing a week. this shouldn’t affect space negatively. the presumption is that desktops aren’t ephemeral, so the whole “missing” thing for a desktop is actually “unknown”

Reply:
@susanev are you thinking about retention settings? this is slightly different -about nodes going missing/deleting missing nodes. the default config for that is to mark as missing after 1 day, i believe, and then delete if missing a week. this shouldn’t affect space negatively. the presumption is that desktops aren’t ephemeral, so the whole “missing” thing for a desktop is actually “unknown”

Reply:
oh yah, totally was. thanks for clarifying.

Reply:
hmm, good idea. ill add it to the issue for all that stuff!

Reply:
flag is actually `--product desktop`

Reply:
does this need more information on how to use the flag? or a link to the config page or something?

Reply:
I was planning on that @susanev but the deploy command listed in https://automate.chef.io/docs/install/#deploy-chef-automate looked significantly different than what @lancewf had demo'd last Friday.

Reply:
the only difference to the deploy command usually used is adding `--product desktop` -- just like we do for bldr and chef server installs

Reply:
we could just provide the exact command here, along with the patch command if they already have automate deployed if we didnt want to link to stuff. also totally fine to come back to this later if yall want 🤷 

ive been fielding what seems like a never-ending stream of how do i add a product questions in success slack so thought maybe we should be a bit more specific